README.md

```diff
@@ -13,6 +13,17 @@
      bundle exec rake db:setup
      rails s
 
+## Testing emails locally
+
+  * `gem install mailcatcher`
```

Comment:
comment: don't add mailcatcher to README (see here: *link to mailcatcher readme*)